applications/portal/frontend/src/components/viewers/detailsViewer/DetailsViewer.tsx

```diff
@@ -0,0 +1,555 @@
+                backgroundColor: 'transparent'
+            }
+        },
+        '& .MuiDivider-root': {
+            backgroundColor: headerBorderColor,
+            marginTop: '0.5rem',
+            marginBottom: '0.5rem'
+        }
+    },
+    menuItemBox: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'space-between',
+        padding: '0.5rem 1rem',
+        '& .MuiMenuItem-root': {
+            padding: 0,
+            '&:hover': {
+                backgroundColor: 'transparent'
+            },
+            '&.Mui-selected': { backgroundColor: 'transparent' }
+        },
+        '& .MuiListItemIcon-root': {
+            minWidth: 'auto'
+        },
+    },
+    searchField: {
+        '& .MuiInputBase-root': {
+            fontSize: '0.75rem',
+            fontWeight: 500,
+            color: sidebarTextColor
+        },
+        '& .MuiInputBase-input': {
+            padding: '0.25rem 1rem'
+        },
+        '& .MuiInput-underline:before': {
+            borderBottom: 'none'
+        },
+        '& .MuiInput-underline:after': {
+            borderBottom: 'none'
+        },
+        '& .MuiInput-underline:hover:not(.Mui-disabled):before': {
+            borderBottom: 'none'
+        }
+    },
+    customTooltip: {
+        '& .MuiTooltip-tooltip': {
+            fontWeight: 400
+        },
+        '&.MuiIconButton-root': {
+            padding: '0.5rem',
+            borderRadius: '0.375rem',
+            marginRight: '0.5rem'
+        }
+    },
+    textBlock: {
+        display: 'flex',
+        flexDirection: 'column',
+        fontSize: '0.75rem',
+        fontWeight: 400,
+        gap: '0.25rem'
+    },
+    populationColor: {
+        display: 'flex',
+        alignItems: 'center',
+        lineHeight: '0.938rem',
+        fontWeight: 400,
+        fontSize: '0.75rem',
+        paddingRight: '0.5rem',
+    },
+
+    square: {
+        width: '0.75rem',
+        height: '0.75rem',
+        borderRadius: '0.1rem',
+    },
+    pagination: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'space-between',
+        '& .MuiTypography-root': {
+            fontSize: '0.75rem',
+            fontWeight: 600,
+            lineHeight: '1rem',
+            color: populationSubTitleColor
+        },
+        '& .MuiPagination-ul li:not(:first-child, :last-child)': {
+            display: 'none'
+        },
+        padding: '0.75rem 1.5rem 1rem 1.5rem'
+    }
+});
+
+const DetailsViewer = (props: {
+    populationName: string | null,
+    populationColor: "#44C9C9"
+}) => {
+    const { populationName, populationColor } = props
+
+    const classes = useStyles();
+    const commonClasses = common();
+    const [tabIdx, setTabIdx] = React.useState(0);
+    const [anchorElMenu, setAnchorElMenu] = React.useState<null | HTMLElement>(null);
+    const [selectedIndex, setSelectedIndex] = React.useState(0);
+    const [category, setCategory] = React.useState('');
```

Comment:
When adding multiple files, this is making all the category dropdowns have the same value which is not correct.
We should have a category for each pdfFile.